action.yml

```diff
@@ -27,23 +30,29 @@ runs:
     env:
       REPO: ${{ inputs.repo }}
     run: |
-      docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
       if [ -z "${{ inputs.tag_list }}" ]; then
-        tag_list=$docker_repo:$REPO,$docker_repo:$(date +%Y%m%d-%H%M)
+        docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        tag_list=()
+        for image_tag in "$REPO" "$REPO-$(date +%Y%m%d-%H%M)"; do
+          tag_list+=("${{ inputs.resitry_url }}/$docker_repo":"$image_tag")
+        done
+        IFS=,
+        echo "::set-output name=taglist::${tag_list[*]}"
       else
         tag_list=${{ inputs.tag_list }}
```

Comment:
Let's also prepend the registry URL to each item in the tag list here. To do this, you'll have to tell folks above how to separate the tags in the description of `tag_list` above (let's go with comma separated)

Reply:
I have added 2 commits to try to resolve this. Could you please check if they are correct? Thanks!